tests/test_diffraction_objects.py

```diff
@@ -232,18 +233,22 @@ def test_diffraction_objects_equality(inputs1, inputs2, expected):
 
 
 def test_dump(tmp_path, mocker):
-    x, y = np.linspace(0, 10, 11), np.linspace(0, 10, 11)
+    x, y = np.linspace(0, 5, 6), np.linspace(0, 5, 6)
```

Comment:
To recollect our memory: we had this invalid value warning:

```
RuntimeWarning: invalid value encountered in arcsin
    print(np.rad2deg(2.0 * np.arcsin(q * pre_factor)))
```

**Problem:**

In our dummy `test_dump` function: 

The code sets arbitrary q values as:

```
# x, y = np.linspace(0, 10, 11), np.linspace(0, 10, 11) sets q
# q = np.asarray(q)
# print(q)
[ 0.  1.  2.  3.  4.  5.  6.  7.  8.  9. 10.]
```

Hence, when we try to convert q to two-theta, using `test.wavelength = 1.54`:

```
print(np.rad2deg(2.0 * np.arcsin(q * pre_factor)))
[  0.          14.07850645  28.37532297  43.14126254  58.70709081
  75.57672216  94.66443973 118.15097974 157.27157928          nan
          nan]
```

We have `nan` values.

**Solution:**

Just simply use the valid q range from 0 to 6, etc.